pyad/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .forward_mode import *
+from .reverse_mode import *
```

Comment:
If we do a star import from both forward and reverse mode, there will be naming conflicts. So, let's just leave the init file empty